.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: Wrangler CI
+
+on:
+  push:
+  pull_request:
+    tags:
+      - v*
+    branches:
+      - 'release/*'
+      - 'master'
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        arch:
+        - amd64
+        - arm64
+    runs-on: org-${{ github.repository_owner_id }}-${{ matrix.arch }}-k8s
+    container: registry.suse.com/bci/golang:1.22
```

Comment:
This container already comes with golang built-in, but we're also installing Go in a later step. How do we know which Go version is being used? Maybe this container should just be a regular ubuntu image (similar to ubuntu-latest, but for self-hosted runners). I kinda prefer to keep the `actions/setup-go` action just because we don't need to update the Go version, but I'm not against removing the step either.

Reply:
Good point - no need to install go

Reply:
Also I think we've discussed this in slack, but the `ubuntu:version` images are too bare-bones. I'd rather use an image that comes with the utilities we need (too bad about mockgen)